assets/javascript/modules/chat-widget-context.test.js

```
@@ -0,0 +1,158 @@
+/**
+ * Tests for chat-widget-context.js
+ */
+
+// Reset modules before each test so module-level state (_navigationListenerActive) is fresh.
+beforeEach(() => {
```

ESLint

[error] 'beforeEach' is not defined.

(no-undef)